recipes/server.rb

```diff
@@ -95,29 +76,11 @@
   returns [0, 42, 127, 3010]
 end
 
-# set the static tcp port
-registry_key static_tcp_reg_key do
-  values [{ name: 'TcpPort', type: :string, data: node['sql_server']['port'].to_s },
-          { name: 'TcpDynamicPorts', type: :string, data: '' }]
-  recursive true
-  notifies :restart, "service[#{service_name}]", :immediately
-end
-
-# If you have declared an agent account it will restart both the
-# agent service and the sql service. If not only the sql service
-if node['sql_server']['agent_account']
-  service agent_service_name do
-    action [:start, :enable]
-  end
-end
-
-service service_name do
-  action [:start, :enable]
-  restart_command %(powershell.exe -C "restart-service '#{service_name}' -force")
-end
-
 # SQL Server requires a reboot to complete your install
 reboot 'sql server install' do
   action :nothing
   reason 'Needs to reboot after installing SQL Server'
 end
+
+include_recipe 'sql_server::configure'
+include_recipe 'sql_server::client'
```

Comment:
As someone using this cookbook I'd really prefer if the client is not installed by default along with the server when the server cookbook is used.